components/speedreader/speedreader_rewriter_service.cc

```diff
@@ -130,7 +130,7 @@ SpeedreaderRewriterService::SpeedreaderRewriterService()
 
 SpeedreaderRewriterService::~SpeedreaderRewriterService() {
   if (file_watcher_) {
-    watch_task_runner_->DeleteSoon(FROM_HERE, file_watcher_);
+    watch_task_runner_->DeleteSoon(FROM_HERE, &*file_watcher_);
```

Comment:
can we just use `file_watcher_.get()` ?

Reply:
`raw_ref::get` returns `T&` rather than `T*` I'm afraid.

Reply:
Ah, my bad. Let me fix that, and check to see if I made such mistake at other places.

Reply:
Done.